### ISSUE TITLE
[Package Request]: Gyroflow

### ISSUE THREAD

**Issue by musm** [microsoft/winget-pkgs#103111]
### How can we help?

I would like someone else to build the manifest.

### Please read and ensure the following

- [X] The installer meets the above requirements

### Please provide the following information

Download Page Url: https://github.com/gyroflow/gyroflow
Publisher: gyroflow
Package Name: gyroflow
Description: Video stabilization using gyroscope data
Package Version: v1.5.0
Installer URL:

**Comment by Trenly** [comment]
[Policy] Zip Binary

**Comment by Trenly** [comment]
@mdanish-kh - Might also need a separate autoresponder for issues?

**Comment by luzpaz** [comment]
Latest  version is 1.5.4
https://github.com/gyroflow/gyroflow/releases/download/v1.5.4/Gyroflow-windows64.zip

**Comment by jo-chemla** [comment]
PR for Gyroflow v1.6.0 has been pushed here https://github.com/microsoft/winget-pkgs/pull/190807

**Comment by Trenly** [comment]
[Policy] Unblocked; Will require `ManifestVersion: 1.9.0` with `ArchiveBinariesDependOnPath: true`

**Comment by jo-chemla** [comment]
Thanks, will push a new PR as soon as Komac supports 1.9.0 manifests file versions!

**Comment by jo-chemla** [comment]
New package submitted here: https://github.com/microsoft/winget-pkgs/pull/197203 with `ArchiveBinariesDependOnPath: true`

**Comment by DandelionSprout** [comment]
The PR seems to have succeeded, so this issue report can presumably be closed as a success.

**Comment by Trenly** [comment]
Close with reason: Resolved; See above comments
